[2022-08-12 04:42:13,883] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_DAG_US_FOOD_NUTRITION.Ingest_from_2016 manual__2022-08-12T04:42:08.092521+00:00 [queued]>
[2022-08-12 04:42:13,901] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_DAG_US_FOOD_NUTRITION.Ingest_from_2016 manual__2022-08-12T04:42:08.092521+00:00 [queued]>
[2022-08-12 04:42:13,902] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-12 04:42:13,903] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-12 04:42:13,904] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-12 04:42:13,918] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): Ingest_from_2016> on 2022-08-12 04:42:08.092521+00:00
[2022-08-12 04:42:13,926] {standard_task_runner.py:52} INFO - Started process 8689 to run task
[2022-08-12 04:42:13,942] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_DAG_US_FOOD_NUTRITION', 'Ingest_from_2016', 'manual__2022-08-12T04:42:08.092521+00:00', '--job-id', '792', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmphblpdpto', '--error-file', '/tmp/tmpgdxo6tky']
[2022-08-12 04:42:13,950] {standard_task_runner.py:80} INFO - Job 792: Subtask Ingest_from_2016
[2022-08-12 04:42:14,119] {task_command.py:371} INFO - Running <TaskInstance: ETL_DAG_US_FOOD_NUTRITION.Ingest_from_2016 manual__2022-08-12T04:42:08.092521+00:00 [running]> on host ae5c2449a624
[2022-08-12 04:42:14,238] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=narapady
AIRFLOW_CTX_DAG_ID=ETL_DAG_US_FOOD_NUTRITION
AIRFLOW_CTX_TASK_ID=Ingest_from_2016
AIRFLOW_CTX_EXECUTION_DATE=2022-08-12T04:42:08.092521+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-12T04:42:08.092521+00:00
[2022-08-12 04:42:22,111] {logging_mixin.py:115} INFO - Successfully uploaded 2016/Table1_2016.xlsx to s3-bucket-raw-usda
[2022-08-12 04:42:23,433] {logging_mixin.py:115} INFO - Successfully uploaded 2016/Table2_2016.xlsx to s3-bucket-raw-usda
[2022-08-12 04:42:24,653] {logging_mixin.py:115} INFO - Successfully uploaded 2016/Table3_2016.xlsx to s3-bucket-raw-usda
[2022-08-12 04:42:25,916] {logging_mixin.py:115} INFO - Successfully uploaded 2016/Table4_2016.xlsx to s3-bucket-raw-usda
[2022-08-12 04:42:27,233] {logging_mixin.py:115} INFO - Successfully uploaded 2016/Table5_2016.xlsx to s3-bucket-raw-usda
[2022-08-12 04:42:28,362] {logging_mixin.py:115} INFO - Successfully uploaded 2016/Table6_2016.xlsx to s3-bucket-raw-usda
[2022-08-12 04:42:29,535] {logging_mixin.py:115} INFO - Successfully uploaded 2016/Table7_2016.xlsx to s3-bucket-raw-usda
[2022-08-12 04:42:30,746] {logging_mixin.py:115} INFO - Successfully uploaded 2016/Table8_2016.xlsx to s3-bucket-raw-usda
[2022-08-12 04:42:30,760] {python.py:173} INFO - Done. Returned value was: None
[2022-08-12 04:42:30,799] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=ETL_DAG_US_FOOD_NUTRITION, task_id=Ingest_from_2016, execution_date=20220812T044208, start_date=20220812T044213, end_date=20220812T044230
[2022-08-12 04:42:30,859] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-12 04:42:30,896] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
